inference/huggingface/text-generation/README.md

```diff
@@ -91,8 +91,9 @@ The DSPipeline class helps to load the model and run inference on it, given thes
 # DeepSpeed HuggingFace Compare
 
 The ds-hf-compare script can be used to compare the text generated outputs of DeepSpeed with kernel injection and HuggingFace inference of a model with the same parameters on a single GPU.
+(p.s. Kernel injection is not supported on CPUs, so you need to disable kernel injection by setting "--use_kernel" as "False" when running on CPUs.)
```

Comment:
Please update this line to say that kernel injection will not be used by default and is only enabled when the `use_kernel` argument is provided.

Reply:
Solved